chrome-extension/open-ai-fetch-interceptor/fetch_interceptor.js

```
@@ -0,0 +1,242 @@
+                    const data = part.substring(5).trim();
+                    if (data && data !== '[DONE]') { // Avoid processing [DONE] again
+                        try {
+                            const jsonPart = JSON.parse(data);
+                            // Accumulate content from final part
+                            if (typeof jsonPart.v === 'string') {
+                                fullResponseText += jsonPart.v;
+                            }
+                            else if (jsonPart.p && jsonPart.p.startsWith('/message/content/parts/') && typeof jsonPart.v === 'string') {
+                                fullResponseText += jsonPart.v;
+                            }
+                            else if (Array.isArray(jsonPart.v)) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)